examples/anime-http/server/functions/trendingAnimeController.ts

```diff
@@ -2,11 +2,13 @@ export default async function (event) {
   const replyPayload = event.payload
   const replyQuery = event.query
   return {
-    reply: [
+    send: [
```

Comment:
This feels wrong to me. We should be replying. Why are you replacing it with `send`?

Reply:
`reply` was removed, @KhudaDad414 can you confirm if it has been removed from the `functions` as well or only the spec support was removed from glee.

Reply:
😱 Removed? Why?